src/UI.elm

```diff
@@ -9,7 +9,7 @@ import Board exposing (Board, Mark(..), toArray)
 import Array exposing (..)
 import Game exposing (Game, GameState(..), retrieveState, board, currentPlayer)
 
-type Msg = TakeTurn Int | Reset
+type Msg = TakeTurn Int | Reset | GameType String
```

Comment:
That makes sense, I had had the same though. I'm not sure I like that game play messages `TakeTurn Int | Reset` are in the same boat as `GameType` selections. Feels like an SRP violation, although that are all UI messages, they have different reasons to be.